text/0000-fabric-private-chaincode-1.0.md

```diff
@@ -439,7 +440,7 @@ In the figure above, the invocation is illustrated with Step 1 - 5.
 The FPC Chaincode processes the invocation according to the implemented chaincode logic.
 While executing, the chaincode can access the World State through `getState` and `putState` operations provided by the FPC Shim. From the chaincode perspective, the arguments of the former, and the output the latter, are plaintext data.
 
-The FPC Shim fetches the state data from the peer and loads it into the enclave, and similarly it stores data by forwarding it to the peer.
+The FPC Shim fetches the state data from the peer and loads it into the enclave.
```

Comment:
I would leave (or rephrase) the deleted line. The shim does not just fetch state.
```suggestion
The FPC Shim stores and fetches the state data from the peer and loads it into the enclave.
```